standards/Docker.md

```diff
@@ -0,0 +1,81 @@
+# Docker
+
+Based on the [Docker guidelines](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/)
+
+## General
+
+1. A `.dockerignore` file SHOULD be used - This will ensure the amount of data sent to the docker daemon will be minimal.
+1. Each container SHOULD have a single concern.
+
+## Dockerfile
+
+1. Multi-line arguments SHOULD be sorted.
+1. The number of layers SHOULD be kept to a minimum.
+1. A single `RUN` command can contain many commands:
+    1. Use `\` and `&&` for multiple commands, with `&&` on the new line:
+
+        ```Dockerfile
+        RUN cmd \
+            && cmd2
+        ```
+
+    1. When adding build time packages, remove them in the same `RUN` command:
+
+        ```Dockerfile
+        RUN apk add --virtual .deps \
+                gcc make \
```

Comment:
Weird indenting. Intentional?

Reply:
It was kind of, to differentiate it from the `&& cmd` lines